### PR TITLE
[FEATURE]/change run time format

### PR DESCRIPTION
**Pull request by serialbandicoot**
Changes proposed in this pull request:
- The Run Time format in the current data docs is showing the format "%Y-%m-%dT%H:%M:%S.%fZ". This PR will change the format to remove the ".%f" - the microseconds, which brings in-line a standard ISO-8601 format and also looks more human readable.

### Definition of Done
Please delete options that are not relevant.

- [ ] My code follows the Great Expectations [style guide](https://docs.greatexpectations.io/en/latest/contributing/style_guide.html?highlight=style%20guide)
- [ ] I have performed a [self-review](https://docs.greatexpectations.io/en/latest/contributing/contribution_checklist.html?highlight=checklist) of my own code
- [ ] I have added [unit tests](https://docs.greatexpectations.io/en/latest/contributing/testing.html#contributing-testing-writing-unit-tests) where applicable and made sure that new and existing tests are passing.
- [ ] I have run any local integration tests and made sure that nothing is broken.

### CODE REVIEW

File: tests/cli/v012/test_init_pandas.py [hunk 1]
```diff
@@ -497,10 +497,7 @@ def test_cli_init_on_new_project_with_broken_excel_file_without_trying_again(
         "- Please check the file and try again or select a different data file."
         in stdout
     )
-    assert (
-        "- Error: Excel file format cannot be determined, you must specify an engine manually."
-        in stdout
-    ) or (
+    assert ("- Error: File is not a recognized excel file" in stdout) or (
```
Comment (serialbandicoot) [inline tests/cli/v012/test_init_pandas.py]: NB: Added this change and same for another test as this was the change I was seeing locally. Happy to reverse it out though if a local issue to resolve. :)

**Comment by talagluck** [comment]
Hi @serialbandicoot - thanks for this! A couple of things:
- It appears that some tests are failing here, and I believe it is due to the code changes in this PR. Are you able to take a look and try and resolve the test failures?
- It appears that your commit history now has commits from both your personal and professional accounts. We have a CLA record for the @serialbandicoot username, with I believe your personal email address. Could you please confirm which one you want here? You may need to rebase to get rid of the commit history for the account you do not want, and if this gets too complicated, it might be easiest to just open a new PR - please let me know how I can help here.

File: tests/cli/v012/test_init_pandas.py (hunk 1 above)
Comment (serialbandicoot) [reply tests/cli/v012/test_init_pandas.py]: No longer relevant as these tests have been reverted.

**Comment by serialbandicoot** [comment]
Hi @talagluck resolved all of the issues, PR over to you.

File: great_expectations/render/renderer/page_renderer.py [hunk 2]
```diff
@@ -352,7 +352,7 @@ def _render_validation_info(cls, validation_results):
             run_time = run_id.get("run_time") or "__none__"
```
Comment (serialbandicoot) [inline great_expectations/render/renderer/page_renderer.py]: Needs to be reviewed for applying strftime...

Comment (serialbandicoot) [reply great_expectations/render/renderer/page_renderer.py]: closing as out-dated

**Review by talagluck — APPROVED** [review]
Thanks for this contribution, @serialbandicoot! LGTM!

**Pull request merged**